addons/survey/models/survey_survey.py

```diff
@@ -17,6 +17,7 @@ class Survey(models.Model):
     and each page can display one or more questions. """
     _name = 'survey.survey'
     _description = 'Survey'
+    _order = 'id DESC'
```

Comment:
I usually prefer create_date desc.
id is something technical and does not really have a functional value in terms of sorting.
(Even if it will be the same end result).